src/helpers/programHelper.js

```diff
@@ -203,14 +203,22 @@ getContributionWithProgramId(program_id, req) {
       nominationResponse = _.isArray(resData[3]) && resData[3].length? _.map(resData[3], obj => obj.dataValues) : [];
       let tableData = [];
     if (collectionList.length) {
-        tableData = _.map(collectionList, (collection) => {
+      let openForContributionCollections = [];
+
+      _.forEach(collectionList, collection => {
+        if (collection.openForContribution === true) {
+          openForContributionCollections.push(collection);
+        }
+      });
+
+        tableData = _.map(openForContributionCollections, (collection) => {
         const result = {};
         // sequence of columns in tableData
         result['Textbook Name'] = collection.name;
         result['Medium'] = collection.medium || '--';
         result['Class'] = collection.gradeLevel && collection.gradeLevel.length ? collection.gradeLevel.join(', ') : '';
         result['Subject'] = collection.subject || '--';
-        result['Number of Chapters'] = collection.chapterCount || '--';
+        result['Number of Chapters'] = collection.chapterCountForContribution || collection.chapterCount || '--';
```

Comment:
Can you check  if currently `chapterCount` is a number or string? 
And let's not keep the fallback value as `--`. it should be 0 or `0`.

Reply:
let's remove `--` from `medium` and `subject` etc... as well